update webview2 debug.md (#1125)
MicrosoftDocs/edge-developer: microsoft-edge/webview2/howto/debug.md

            "Path": "%path%;e:/path/to/your/application/location; "
        },
        "useWebView": true,

>         // The following two lines setup source path mapping

Reviewer: ```suggestion
        // The following two lines setup source path mapping, where `url` is the start page of your application, and `webRoot` is the top level directory with all your code files.
```

        "url": "file:///${workspaceFolder}/path/to/your/toplevel/FOO.HTML",
        "webRoot": "${workspaceFolder}/path/to/your/assets"
    ```  
        > [!NOTE]

>     > Because Visual Studio Code source path mapping requires the URL configuration setup your application will receive a command line parameter on startup. If unneeded you can safely ignore this parameter.  

Reviewer: ```suggestion
    > Because Visual Studio Code source path mapping requires the URL, your application now receives a command line parameter when it starts. You can safely ignore this parameter if needed.  
```

    :::image-end:::
    
    > [!NOTE]

>     > Because Visual Studio Code source path mapping requires the URL configuraiton setup your application will recive a command line parameter on startup.  If uneeded you can safely ignore this parameter.  

Reviewer 2: receive*, not needed* or unneeded* I think?

Author: Spelling fixed.
